src/lib/rng/jitter_rng/jitter_rng.h

```diff
@@ -0,0 +1,44 @@
+/*
+* CPU Jitter Random Number Generator
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_JITTER_RNG_H_
+#define BOTAN_JITTER_RNG_H_
+
+#include <botan/rng.h>
+
+namespace Botan {
+
+namespace {
+
+struct Jitter_RNG_Internal;
+
+}
```

Comment:
```suggestion
struct Jitter_RNG_Internal;
```

... as already concluded by @Delta-dev-99 [here](https://github.com/randombit/botan/pull/4325#issuecomment-2395046695).